django/db/backends/utils.py

```diff
@@ -65,14 +66,27 @@ def callproc(self, procname, params=None, kparams=None):
                 return self.cursor.callproc(procname, params, kparams)
 
     def execute(self, sql, params=None):
+        return self._execute_with_wrappers(sql, params, False)
+
+    def executemany(self, sql, param_list):
+        return self._execute_with_wrappers(sql, param_list, True)
+
+    def _execute_with_wrappers(self, sql, params, many):
+        executor = self._execute if not many else self._executemany
+        context = dict(connection=self.db, cursor=self)
```

Comment:
`{'connection': self.db, 'cursor': self}` is faster and imo clearer. `dict(foo=bar)` actually goes through two dict creation steps: first it creates the keyword args dict, then it finds the global name 'dict' (which could refer to anything), then it calls that name which creates the final dict.

Reply:
Yes, I like using `dict` when the keys are really symbols, but you're right.